instrumentation/opentelemetry/init.go

```diff
@@ -111,6 +112,11 @@ func makeExporterFactory(cfg *config.AgentConfig) func() (sdktrace.SpanExporter,
 			}
 		}
 
+		if cfg.Reporting.GetEnableGrpcLoadbalancing().GetValue() {
```

Comment:
@prodion23 do we not need to add default value in https://github.com/hypertrace/goagent/blob/4823a3e6efd3dc1fc622331ec6b8a7c7e27e9716/config/defaults.go#L34?

Reply:
Very good catch.

Reply:
Thanks for catching that, will fix